app/graphql/types/enums/delete_membership_type_enum.rb

```diff
@@ -0,0 +1,11 @@
+module Types
+  module Enums
+    DeleteMembershipTypeEnum = GraphQL::EnumType.define do
```

Comment:
We should try to keep these enums 1 to 1 with the real enum in the model. I.E. rename this to be the same as whatever the enum is called in the User model. That way it's a bit easier to follow, especially for others viewing this code.